### DISCUSSION TITLE
What is the capital of 🇨🇭?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1006] [Galaxy Trivia] [answered]
**Bonus:** Which country produces the most 🐮🥛?

**Comment by peterbaccaro** [comment]
The capital of Switzerland is **Bern**.

The **USA** is the country that produces the most cow milk.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/63c8d47c-bee6-4365-a752-4f23832d2a94)

https://beef2live.com/story-ranking-countries-produce-cow-milk-fao-0-215760

**Comment by thuongtruong1009** [comment, marked as answer]
The capital of Switzerland, represented by the country code 🇨🇭, is Bern. It is located in the central part of the country and serves as the administrative center of Switzerland.

As for the country that produces the most cow's milk, as of my knowledge cutoff in September 2021, the country that holds that distinction is India. India has a large dairy industry and is one of the leading producers of cow's milk in the world. However, please note that agricultural production can vary over time, so it's advisable to check the most recent data for the latest information on milk production.
